zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/GoogleCloudStorageNotebookRepo.java

```diff
@@ -38,11 +38,18 @@
   private static final Logger LOG = LoggerFactory.getLogger(GoogleCloudStorageNotebookRepo.class);
 
   private final ZeppelinConfiguration conf;
+  private final int GCSNotebookMaxRetriesToSave;
   private final GoogleCloudStorage cloudStorage;
 
   public GoogleCloudStorageNotebookRepo(ZeppelinConfiguration conf) throws IOException {
     this.conf = conf;
+    this.GCSNotebookMaxRetriesToSave = conf.getInt(
```

Comment:
What is the plan if that value isn't provided?

Reply:
We are providing a default value in ZeppelinConfiguration.java  . It is currently set to 30.